tests/composite_pk/test_models.py

```diff
@@ -155,3 +155,9 @@ def test_permissions(self):
         self.assertEqual(4, token.permission_set.count())
         self.assertEqual(4, user.permission_set.count())
         self.assertEqual(4, comment.permission_set.count())
+
+    def test_refresh_foreign_object(self):
+        comment = Comment.objects.get(pk=self.comment_1.pk)
+        comment.user = None
```

Comment:
This is probably a better assertion so that we don't get sidetracked by the issue of setting a nonexistent primary key. I didn't realize when sketching out the draft test that the `to_fields` on the `User` field included fields that participate in the CompositePrimaryKey of the `Comment` model.
```suggestion
        comment.user = self.user_2
```